Makefile

```diff
@@ -16,7 +16,7 @@ help:
 INSTALL=_build/install/default
 BUILD=_build/default
 SHELL:=/bin/bash
-TIMEOUT=90.0
+TIMEOUT=300.0
```

Comment:
Why this?

Reply:
The Travis-CI tests were timing out (I think it was the grundlagen test, but I forget); that's why I increased the timeout.  It might be better to change things so that at least for the opam build, only a subset of the tests were run, since elpi is built whenever anything it depends-upon, is changed.

Reply:
Grundlagen is an important test. On my machine it spends ~ 8 seconds in parsing and about 20 in execution. I'll try it myselft to see if there is a camlp5 regression.

Reply:
Here I see no difference between master and this PR wrt speed, so please revert this change

Reply:
Done.  I will note, that it wasn't for the difference in speed, but rather, so that CI tests wouldn't timeout.